tests/sentry/api/endpoints/test_project_plugin_details.py

```diff
@@ -76,6 +76,10 @@ def test_simple(self):
             project=project,
         ).value is True
 
+        # Testing the Plugin
+        response = self.client.post(url, {'test': True})
+        assert response.status_code == 200, (response.status_code, response.content)
```

Comment:
can you add an assertion about the content? or even just that the plugin's test_configuration was called? you might want to use something like `mock.patch` to control either the message or assert that that method was called.